pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
No, I think you misunderstood. Add the require to `lib/sysfs.rb`. Not in the gemspec. We would add the `spec.add_dependency` as you already did. I have used this approach for some other gems and it seems to be fine.